CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Current Master
 
+- Added ShapeShift to the transaction history
+- Added PUB.KEY too ShapeShift request for affiliate program
```

Comment:
Make this prettier for normal people. `Added affiliate key to shapeshift requests`, maybe?